### PR TITLE
[IE TESTS] Add Interpolate single layer test

### PR DESCRIPTION
**Pull request by iefode**
CC @echuraev @vgavrilo @lazarevevgeny

### CODE REVIEW

File: ngraph/src/ngraph/op/interpolate.hpp [hunk 1]
```diff
@@ -47,6 +47,20 @@ namespace ngraph
                 // specify the number of pixels to add to the end of the image being interpolated.
                 // This addition of pixels is done before interpolation calculation.
                 std::vector<size_t> pads_end;
+
+                InterpolateAttrs(){};
```
Comment (ilya-lavrenov) [inline ngraph/src/ngraph/op/interpolate.hpp:51]: `;` is not needed at the end

Comment (mikhail-treskin) [inline ngraph/src/ngraph/op/interpolate.hpp:51]: What was the reason to update v0 operation but not v3?

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    std::vector<float> data(targetShape.size());
+    for (int i = 0 ; i < targetShape.size(); ++i) {
+        data[i] = targetShape[i];
+    }
+    auto secondaryInput = ngraph::builder::makeConstant(ngraph::element::Type_t::i64, {axes.size()}, data, false);
+
+    ngraph::op::InterpolateAttrs interpolateAttributes{axes, mode2String(mode), alignCorners, antialias, padBegin, padEnd};
+
+    auto interpolate = std::make_shared<ngraph::opset3::Interpolate>(params[0], secondaryInput, interpolateAttributes);
```
Comment (mikhail-treskin) [inline inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp:86]: I'm a little bit confused about how v3 operation constructed with attributes from v0?

File: inference-engine/tests/functional/plugin/shared/include/single_layer_tests/interpolate.hpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+namespace LayerTestsDefinitions {
+
+typedef std::tuple<
+        std::set<size_t>,                            // Axes
+        ngraph::op::Interpolate::InterpolateMode,    // InterpolateMode
```
Comment (mikhail-treskin) [inline inference-engine/tests/functional/plugin/shared/include/single_layer_tests/interpolate.hpp:20]: please check v3 operation - it contains some additional parameters which are not used in current implementation

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    ngraph::op::Interpolate::InterpolateMode mode;
+    std::tie(axes, mode, alignCorners, antialias, padBegin, padEnd) = interpolateParams;
+
+    if (targetShape.size() != axes.size()) {
+        THROW_IE_EXCEPTION << "Target shape size: " << targetShape.size() << " is not equal Axes shapes: " <<  axes.size();
+    }
+
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    std::vector<float> data(targetShape.size());
```
Comment (mikhail-treskin) [inline inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp:78]: this copying looks strange for me. Is it done just to fit makeConstant interface?

I propose to use `Constant(const element::Type& type, Shape shape, const std::vector<T>& values)` constructor diretcly and get rid of creating additional vector `data`

File: ngraph/src/ngraph/op/interpolate.hpp (hunk 1 above)
Comment (iefode) [reply ngraph/src/ngraph/op/interpolate.hpp]: Fixed

Comment (iefode) [reply ngraph/src/ngraph/op/interpolate.hpp]: Fixed

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp (hunk 2 above)
Comment (iefode) [reply inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp]: Fixed

File: inference-engine/tests/functional/plugin/shared/include/single_layer_tests/interpolate.hpp (hunk 3 above)
Comment (iefode) [reply inference-engine/tests/functional/plugin/shared/include/single_layer_tests/interpolate.hpp]: Fixed

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp (hunk 4 above)
Comment (iefode) [reply inference-engine/tests/functional/plugin/shared/src/single_layer_tests/interpolate.cpp]: Fixed

File: ngraph/src/ngraph/op/interpolate.hpp [hunk 5]
```diff
@@ -136,18 +144,38 @@ namespace ngraph
                     NearestMode nearest_mode = NearestMode::round_prefer_floor;
                     // a flag that specifies whether to perform anti-aliasing. default is `false`
                     bool antialias = false;
```
Comment (ilyachur) [inline ngraph/src/ngraph/op/interpolate.hpp:146]: You can remove this default values if you set their in the constructor.

Comment (iefode) [reply ngraph/src/ngraph/op/interpolate.hpp:146]: Ok, I will do it in next pr

Comment (iefode) [reply ngraph/src/ngraph/op/interpolate.hpp:146]: FIxed: https://github.com/openvinotoolkit/openvino/pull/1484/

**Pull request merged**